crates/matrix-sdk/src/media.rs

```diff
@@ -60,6 +60,11 @@ pub struct Media {
 pub struct MediaFileHandle {
     /// The temporary file that contains the media.
     file: NamedTempFile,
+    /// An intermediary temporary directory used in certain cases.
+    ///
+    /// Only stored for its `Drop` semantics.
+    #[allow(dead_code)]
```

Comment:
Oh, maybe I fudged my previous suggestion. `dead_code` doesn't need to be silenced on variables that start with `_`.

```suggestion
```

Reply:
Oh, I thought the underscore was just a rust convention, fixed it 👍